diff --git a/draggableviews.module b/draggableviews.module
index d724fa1..60efdb9 100644
--- a/draggableviews.module
+++ b/draggableviews.module
@@ -140,7 +140,8 @@ function draggableviews_form_alter(&$form, FormStateInterface $form_state, $form
   $form['#attached']['library'][] = 'draggableviews/draggableviews';
 
   // Remove default submit button.
-  $form['actions']['submit']['#access'] = FALSE;
+  // stop doing this
+  //$form['actions']['submit']['#access'] = FALSE;
 
   if (\Drupal::currentUser()->hasPermission('access draggableviews')) {
     // Create draggableviews save order button.
